### DISCUSSION TITLE
没有文件：monaco.d.ts，甚至没有monaco-editor文件夹

### DISCUSSION THREAD

**Discussion by input111** [DevToys-app/DevToys#913] [Q&A]
严重性	代码	说明	项目	文件	行	禁止显示状态
错误		Build:找不到文件“E:/.0using/桌面/DevToys-0.7.5.0/src/dev/impl/DevToys.MonacoEditor/monaco-editor/monaco.d.ts”。	DevToys.MonacoEditor	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\otherScriptsToBeOrganized.ts	1	

因此，导致无法运行：
严重性	代码	说明	项目	文件	行	禁止显示状态
错误		Build:找不到命名空间“monaco”。	DevToys.MonacoEditor	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\otherScriptsToBeOrganized.ts	5	
错误		Build:找不到命名空间“monaco”。	DevToys.MonacoEditor	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\otherScriptsToBeOrganized.ts	6	
错误		Build:找不到命名空间“monaco”。	DevToys.MonacoEditor	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\otherScriptsToBeOrganized.ts	7	
错误		Build:找不到名称“monaco”。	DevToys.MonacoEditor	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\otherScriptsToBeOrganized.ts	12	
错误		Build:找不到命名空间“monaco”。	DevToys.MonacoEditor	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\otherScriptsToBeOrganized.ts	23	
错误		Build:找不到命名空间“monaco”。	DevToys.MonacoEditor	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\otherScriptsToBeOrganized.ts	55	
错误		Build:找不到命名空间“monaco”。	DevToys.MonacoEditor	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\otherScriptsToBeOrganized.ts	80	
错误		Build:找不到命名空间“monaco”。	DevToys.MonacoEditor	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\otherScriptsToBeOrganized.ts	95	
错误		Build:找不到命名空间“monaco”。	DevToys.MonacoEditor	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\otherScriptsToBeOrganized.ts	110	
错误		Build:找不到命名空间“monaco”。	DevToys.MonacoEditor	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\otherScriptsToBeOrganized.ts	116	
错误		Build:找不到命名空间“monaco”。	DevToys.MonacoEditor	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\otherScriptsToBeOrganized.ts	117	
错误		Build:找不到名称“monaco”。	DevToys.MonacoEditor	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\otherScriptsToBeOrganized.ts	124	
错误		Build:找不到名称“monaco”。	DevToys.MonacoEditor	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\otherScriptsToBeOrganized.ts	135	
错误		Build:找不到名称“monaco”。	DevToys.MonacoEditor	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\otherScriptsToBeOrganized.ts	141	
错误		Build:找不到名称“monaco”。	DevToys.MonacoEditor	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\otherScriptsToBeOrganized.ts	157	
错误		Build:找不到文件“E:/.0using/桌面/DevToys-0.7.5.0/src/dev/impl/DevToys.MonacoEditor/monaco-editor/monaco.d.ts”。	DevToys.MonacoEditor	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\registerCompletionItemProvider.ts	1	
错误		Build:找不到名称“monaco”。	DevToys.MonacoEditor	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\registerCompletionItemProvider.ts	5	
错误		Build:找不到文件“E:/.0using/桌面/DevToys-0.7.5.0/src/dev/impl/DevToys.MonacoEditor/monaco-editor/monaco.d.ts”。	DevToys.MonacoEditor	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\updateSelectedContent.ts	1	
错误		Build:找不到命名空间“monaco”。	DevToys.MonacoEditor	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\updateSelectedContent.ts	5	
错误		Build:找不到命名空间“monaco”。	DevToys.MonacoEditor	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\updateSelectedContent.ts	6	
错误		Build:找不到命名空间“monaco”。	DevToys.MonacoEditor	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\updateSelectedContent.ts	7	
错误		Build:找不到名称“monaco”。	DevToys.MonacoEditor	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\updateSelectedContent.ts	17	
错误		“CompileXaml”任务返回了 false，但未记录错误。	DevToys			
错误		Cannot resolve Assembly or Windows Metadata file 'E:\.0using\桌面\DevToys-0.7.5.0\bin\Debug\x64\DevToys.MonacoEditor\DevToys.MonacoEditor.winmd'	DevToys	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys\DevToys.csproj		
错误	TS6053	(TS) 找不到文件“E:/.0using/桌面/DevToys-0.7.5.0/src/dev/impl/DevToys.MonacoEditor/monaco-editor/monaco.d.ts”。	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\tsconfig.json	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\otherScriptsToBeOrganized.ts	1	活动
错误	TS2503	(TS) 找不到命名空间“monaco”。	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\tsconfig.json	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\otherScriptsToBeOrganized.ts	5	活动
错误	TS2503	(TS) 找不到命名空间“monaco”。	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\tsconfig.json	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\otherScriptsToBeOrganized.ts	6	活动
错误	TS2503	(TS) 找不到命名空间“monaco”。	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\tsconfig.json	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\otherScriptsToBeOrganized.ts	7	活动
错误	TS2304	(TS) 找不到名称“monaco”。	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\tsconfig.json	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\otherScriptsToBeOrganized.ts	12	活动
错误	TS2503	(TS) 找不到命名空间“monaco”。	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\tsconfig.json	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\otherScriptsToBeOrganized.ts	23	活动
错误	TS2503	(TS) 找不到命名空间“monaco”。	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\tsconfig.json	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\otherScriptsToBeOrganized.ts	55	活动
错误	TS2503	(TS) 找不到命名空间“monaco”。	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\tsconfig.json	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\otherScriptsToBeOrganized.ts	80	活动
错误	TS2503	(TS) 找不到命名空间“monaco”。	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\tsconfig.json	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\otherScriptsToBeOrganized.ts	95	活动
错误	TS2503	(TS) 找不到命名空间“monaco”。	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\tsconfig.json	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\otherScriptsToBeOrganized.ts	110	活动
错误	TS2503	(TS) 找不到命名空间“monaco”。	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\tsconfig.json	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\otherScriptsToBeOrganized.ts	116	活动
错误	TS2503	(TS) 找不到命名空间“monaco”。	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\tsconfig.json	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\otherScriptsToBeOrganized.ts	117	活动
错误	TS2304	(TS) 找不到名称“monaco”。	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\tsconfig.json	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\otherScriptsToBeOrganized.ts	124	活动
错误	TS2304	(TS) 找不到名称“monaco”。	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\tsconfig.json	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\otherScriptsToBeOrganized.ts	135	活动
错误	TS2304	(TS) 找不到名称“monaco”。	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\tsconfig.json	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\otherScriptsToBeOrganized.ts	141	活动
错误	TS2304	(TS) 找不到名称“monaco”。	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\tsconfig.json	E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys.MonacoEditor\ts-helpermethods\otherScriptsToBeOrganized.ts	157	活动
警告		命令“...”已退出，代码为 128。	DevToys			
警告		命令“...”已退出，代码为 128。	DevToys			
警告		Directory E:\.0using\桌面\DevToys-0.7.5.0\src\dev\impl\DevToys is not in a Git repository. Cannot determine Git repository root.	DevToys			
警告		已启用包签名，但未指定指纹或证书路径。	DevToys.Startup	C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Microsoft\VisualStudio\v17.0\AppxPackage\Microsoft.AppXPackage.Targets	875

**Comment by JIMZX** [comment]
请去下载一个MonacoEditor的代码，解压放到对应目录就可以了
